### DISCUSSION TITLE
Use an IAM User or IAM Role to access Storage Lens dashboards

### DISCUSSION THREAD

**Discussion by ghost** [learndevopsonline/learndevopsonline#1503] [Q&A]
hi @r-devops 

How to  IAM Role to access Storage Lens dashboards.Exactly I dont know
Could you please help me
![Screenshot 2023-10-04 144512](https://github.com/learndevopsonline/learndevopsonline/assets/146338851/cae5a6ba-98e2-4019-a3a1-a3c1c6dcfeb9)

**Comment by sreedharm07** [comment]
Hi @Chimtamreddy 
there is a service called as IAM. in that you can see user option, create user and give username and in that give permissions for --s3:ListStorageLensConfigurations
s3:GetStorageLensConfiguration
s3:GetStorageLensDashboard

then with that credentials log in and  open the storage lence you can view it.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/46fc175e-6ff4-43a6-8dda-887b2cea6caa)

**Reply by ghost** [reply]
**ok,thank you** i will try ,you know how to create **terraform backend account**.Could you please help me.

**Reply by sreedharm07** [reply]
backend account means, you mean how to upload terraform state file to s3 bucket ?

**Reply by ghost** [reply]
how to add storage s3 bucket

**Reply by sreedharm07** [reply]
Go to **s3 service --> buckets --> create bucket -->** then give your bucket name and other specifications and create 

![image](https://github.com/learndevopsonline/learndevopsonline/assets/137472068/fab99e19-d9b6-4901-8c1a-bd5001004f30)

**Comment by ghost** [comment]
I need Account Snapshot How to get
![Screenshot 2023-10-04 154905](https://github.com/learndevopsonline/learndevopsonline/assets/146338851/9b94444e-a97a-4ac8-93dd-39e49ab78b43)

**Comment by ghost** [comment]
Which one choose Recomonded or I want creat new IAM User
![Screenshot 2023-10-04 155920](https://github.com/learndevopsonline/learndevopsonline/assets/146338851/2658d6f8-232f-4f7e-adb0-5667df91f323)

**Reply by sreedharm07** [reply]
i think you are going in a wrong way.
there is no need to create snapshot and also no need to view storage lens. we just want to create one s3 bucket and give configuration in terraform(which you already given) to store our state file in that bucket.

**Reply by ghost** [reply]
ok thank you bro
